pages/design/ui-patterns/button-usage.mdx

```diff
@@ -0,0 +1,129 @@
+  </Box> 
+</Flex>
+
+### Use Text Descriptions
+In certain scenarios, buttons require supporting text to give the user more context. Use text labels sparingly, as buttons should generally be able to be understood on their own.
+
+<Flex>
+  <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='green.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/52501400-1b522300-2b95-11e9-8c47-71af68cfaeec.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='green.5'>Do:</Text> Keep text brief, specific, and descriptive
+  </Box> 
+   <Box width='100%' mr='2'>
+    <BorderBox border='1px solid' borderColor='red.5' mb="2">
+      <img src="https://user-images.githubusercontent.com/24916540/53110379-cbfee180-3508-11e9-8762-04e3c8979afb.png"/>
+    </BorderBox>
+    <Text fontWeight='bold' color='red.5'>Don't:</Text> Use unnecesary, redundant text descriptions
+  </Box> 
+</Flex>
+
+
+### Use Octicons to Clarify Actions
+Use octicons in conjunction with buttons to clarify an action if text alone is not enough to explain its functionality. Octicons can also be used to signal an interaction, for example, the `triangle-down` octicon to open a dropdown.
+
+In some cases, Octicons are used to create an identifiable visual indicator for a common action or paradigm. For example, the repo icon is used next to repo links, the new repo button, and repeated in the new repo flow. This repetition and continuity helps people learn and identify our visual language.
+
+### Use Button Groups to Organize Similar Functionality  
+Organize buttons together in button groups to show their similar functionality. Keep in mind that only default and outline buttons can be used as button groups. This modifier can also be used as a filtering pattern as well.
+
+### Using Social Proof
+Counters can be added as numerical aid in appropriate scenarios. They should generally only be used with “small” buttons, and only be used where a it is providing valuable context.
+
+
+## Hierarchy and Placement
+
+### Use Size to Modify Button Hierarchy
+Increase button size to bring prominence to buttons that have major impact. Decrease button size when space is limited and the action is of lesser significance. Use this modifier sparingly, as layouts should generally have only one large button. 
+
+
+### Limit Primary Button Usage
+Only use one primary button on the page, whenever possible, to indicate its emphasis relative to other actions. Primary buttons have  top priority in visual hierarchy, and using too many of them on a single view dilutes their effectiveness.
+
+### Always Align Primary Buttons to the Outermost Edge 
+When a secondary button is placed in conjunction with a primary button, the primary button should always be placed on the outermost alignment, or on top and extended full-width if stacked.
+
+<Flex>
+  <Box width='100%' mr='2'>
```

Comment:
I would advise against using the comment buttons as an example since the version you've shown here is no longer in production and incorrectly uses title case, not sentence case in the example. I would use the close pull request example with the primary and secondary layout:

<img width="396" alt="screenshot 2019-02-22 09 53 12" src="https://user-images.githubusercontent.com/334891/53250140-cd0a4d00-3687-11e9-85d0-4b0ba2bd6027.png">